packages/astro/src/runtime.ts

```diff
@@ -345,7 +346,7 @@ async function createSnowpack(astroConfig: AstroConfig, options: CreateSnowpackO
 
   // Make sure that Snowpack builds our renderer plugins
   const rendererInstances = await configManager.buildRendererInstances();
-  const knownEntrypoints: string[] = [];
+  const knownEntrypoints: string[] = ['astro/dist/internal/__astro_component.js'];
```

Comment:
Yeah I just saw this warning. It didn't come up when I was working on the PR because of the workspace stuff :(